src/vi/hentaivn/src/eu/kanade/tachiyomi/extension/vi/hentaivn/HentaiVN.kt

```diff
@@ -43,7 +43,7 @@ class HentaiVN : ParsedHttpSource(), ConfigurableSource {
         Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
     }
 
-    private val defaultBaseUrl = "https://henvn.tv"
+    private val defaultBaseUrl = "https://hentaihvn.tv"
```

Comment:
old url redirects to `https://hentaivn.icu` for me?

Reply:
hentaivn.icu is like a scapegoat for hentaihvn.tv (which is the main website of hentaivn)
when you search for hentaivn, you will only see .icu so that the main website can stay longer before being blocked
if you choose login/register in .icu website, it will redirect you to hentaihvn.tv